src/Views/gagepage.html

```diff
@@ -195,10 +195,28 @@ <h4 class="modal-title">StreamStats Gage Page</h4>
                 </div>
             </div>
 
-            <!-- Gage Analysis Plots Tab-->
+            <!-- Gage Analysis Plots Tab -->
             <div ng-if="vm.SelectedTab == 2" style="position: relative;">
                 <div style="padding-top: 10px;">
-                    Gage Analysis Plots will go here
+                    <div class="panel panel-default">
+                        <div class="panel-heading">Gage Information</div>
+                            <table class="table">
```

Comment:
For this table, see if you can copy the styling (the column widths and the the background colors of the rows) that is seen on the "Gage Information" tab:
![image](https://user-images.githubusercontent.com/40237491/210883471-adcf76ae-faed-494f-a918-4056a1601486.png)

Reply:
Copied the colors and styling. Seems like column widths are correlated with the length of the content in them.